posts/blender_w3d_plugin/blender_w3d_plugin.md

```diff
@@ -0,0 +1,64 @@
+---
+slug: "blender-w3d-plugin-2018"
+title: "A Blender W3D Plugin"
+summary: "Import & Exporting W3D files into Blender"
+author: "Stephan Vedder"
+date: "2019-11-05"
+---
+
+As we've already announced in our latest post, we're working on a **W3D Blender Plugin**. We strongly believe that modding is a big part of SAGE being such a succesful engine over the years, that's why we want to contribute tools back to the community.
+
+You probably already know that the Westwood 3D (W3D) format is the 3D format used in the first games that were created with the SAGE engine (later on replaced by a serialized version called W3X). One big issue with that format used to be that there are very little tools that allow authoring of these files. The offical modding SDKs contained plugins for old Autodesk 3dsmax versions (which were recent at the time). The issue is that Autodesk 3dsmax is a commercial tool and the offical plugin is not available for current versions anymore. Apart from this it was lacking some features the community desperatly needs, like the import of compressed animations.
+
+As a conclusion of this we decided to write an opensource Blender Plugin, which allows to export and import W3D files. Our existing W3D code from OpenSage was a big advantage, since it helped us getting a strong understanding of the format itself.
```

Comment:
```suggestion
As the result of this we decided to write an open source Blender plugin, which adds support for exporting and importing of W3D files. Our existing W3D code from OpenSAGE was a big advantage, since it serves as the most comprehensive documentation available for the format.
```